### PR TITLE
Added CONTRIBUTING.md, updated README.md

### LINKED ISSUE #138: feature: Add docs contributing guidelines
There are two types of contrubutions to this repo: 
1. adding yourself (according to chapter directions).
2. updating the repo to make it a better contributor experience.

We've been successful with the docs for the first. Let's improve the docs for the second.

To do this, we need to explain how to run the project locally: https://github.com/open-sauced/intro/blob/main/CONTRIBUTING.md.

I'm in favor of adding this CONTRIBUTING.md file to our repo and linking in our readme with an explanation that if you're interested in contributing to improve this course, please create an issue and refer to this guide.

### PR DESCRIPTION
**Pull request by codewithjazzy**
## Description
This PR adds the CONTRIBUTING.md file and links to it in the README.md

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
Closes #138 

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [x] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?
Bart Simpson exclaims "Yes!" and fist pumps.

### CODE REVIEW

**Comment by adiati98** [comment]
Hey @codewithjazzy,
Thank you for your PR!

We have merged a PR before yours. Please [resolve the conflicts](https://github.com/open-sauced/guestbook#keeping-branch-up-to-date-and-resolving-merge-conflicts) before we can merge yours.

And please let us know if you need any help 😊

**Comment by CBID2** [comment]
Thanks for your Pull Request @codewithjazzy! We've merged in some new PRs since you submitted. Can you merge the latest changes and resolve conflicts? If you need some help with that, check out this post: [Keeping Your Branch Up to Date and Handling Merge Conflicts While Waiting for PR Reviews](https://dev.to/opensauced/keeping-your-branch-up-to-date-and-handling-merge-conflicts-while-waiting-for-pr-reviews-3b3h).

**Review by CBID2 — APPROVED** [review]
Looks good @codewithjazzy

**Pull request merged**
